<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RiggingTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Creation</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>731</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>489</width>
         <height>729</height>
        </rect>
       </property>
       <widget class="QPushButton" name="specialFk">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>650</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>specialFK</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>480</y>
          <width>231</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>AddJointToJoints</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>The number of Joints</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numJoints"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addJointsButton">
               <property name="text">
                <string>Add Joints</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>461</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>RigControllers</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>421</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="makeCircleButton">
            <property name="text">
             <string>Circle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="makeSquareButton">
            <property name="text">
             <string>Square</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="makeCubeButton">
            <property name="text">
             <string>Cube</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>461</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Facial</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>51</y>
           <width>401</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="loadBaseButton">
            <property name="text">
             <string>BaseHead</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="faceCurveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>CreateFaceCurve</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="facialJointButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>JointCreate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Control</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
